gdnative-derive/src/native_script.rs

```diff
@@ -153,8 +168,9 @@ fn parse_derive_input(input: &DeriveInput) -> Result<DeriveData, syn::Error> {
         .attrs
         .iter()
         .find(|a| a.path.is_ident("register_with"))
-        .map(|attr| attr.parse_args::<Path>())
-        .transpose()?;
+        .map(|attr| attr.parse_args::<CallbackArgs>())
+        .transpose()?
+        .map(|cba| cba.paths);
 
```

Comment:
Does this handle the case of having multiple `register_with` attributes on the same type? Should an empty `register_with` attribute be considered an error?